File: orden/BubbleSort.java
class BubbleSort{
    public static void main(String[] args){
        int[] list = {8,3,1,19,14};
        int temp;
        for(int k : list){
            System.out.print(k+" ");
        }
        System.out.println();
        for(int i=5; i>0; i--){
            for(int j=0; j<(i-1); j++){
                if(list[j]>list[j+1]){
                    temp = list[j];
                    list[j] = list[j+1];
                    list[j+1] = temp;
                }
            }
        }
        for(int k : list){
            System.out.print(k+" ");
        }
    }
}

File: orden/BubbleSort.py
import random
import time
#lista = [7,3,10,1,9]
lista = random.sample(range(1,1001),1000)
#print(lista)
inicio = time.time()
for i in range(len(lista),0,-1):
    for j in range(0,(i-1)):
        if lista[j] > lista[j+1]:
            lista[j], lista[j+1] = lista[j+1], lista[j]
fin = time.time()
#print(lista)
print(f"Tiempo con BubbleSort: {fin-inicio}")
